core/kazoo_globals/src/kz_nodes.erl

```diff
@@ -339,7 +339,11 @@ maybe_print_roles(Roles) ->
 -spec print_role({ne_binary(), kz_json:object()}) -> 'ok'.
 print_role({<<"Dispatcher">>, Data}) ->
     Groups = kz_json:get_json_value(<<"Groups">>, Data, kz_json:new()),
-    kz_json:foreach(fun print_dispatcher/1, Groups);
+    Keys = lists:sort(kz_json:get_keys(Groups)),
+    lists:foreach(fun(Group) ->
+                          GData = kz_json:get_json_value(Group, Groups),
+                          print_dispatcher({Group, GData})
+                  end, Keys);
```

Comment:
the keys for the dispatcher are not sorted, we get those, sort them and the elements by that order